mupdf-sys/build.rs

```diff
@@ -305,21 +293,20 @@ fn build_libmupdf() {
         if cfg!(not(feature = "js")) {
             cl_env.push("/DFZ_ENABLE_JS#0".to_string());
         }
-        let d = cc::windows_registry::find(target.as_str(), "devenv.com")
+        let d = cc::windows_registry::find(target.as_str(), "msbuild.exe")
             .unwrap()
             .args(&[
                 "platform\\win32\\mupdf.sln",
-                "/build",
-                &format!("{}|{}", profile, msvc_platform),
-                "/project",
-                "libmupdf",
+                "/target:libmupdf",
+                &format!("/p:Configuration={}", profile),
+                &format!("/p:Platform={}", msvc_platform),
             ])
             .current_dir(&build_dir)
             .env("CL", cl_env.join(" "))
```

Comment:
Does MSBuild uses this env var? I'm not sure.

Reply:
The env var should directly affect the compiler (CL.exe) so should be independent of how it gets invoked. Also a quick test seems to indicate that disabling support for some formats does indeed work.

Reply:
Thanks for confirming!